cmd/esc/cli/env_tag_mv.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2024, Pulumi Corporation.
+
+package cli
+
+import (
+	"context"
+	"errors"
+
+	"github.com/pulumi/esc/cmd/esc/cli/style"
+	"github.com/spf13/cobra"
+)
+
+func newEnvTagMvCmd(env *envCommand) *cobra.Command {
+	var utc bool
+
+	cmd := &cobra.Command{
+		Use:   "mv [<org-name>/]<environment-name> <name> [<newName>] <value>",
```

Comment:
I would remove the `<value>` bit here and make this command purely a "rename" command:

`mv [<org-name>/]<environment-name> <name> <newName>`

Reply:
ah ok and keep the value updates under the tag "set" command?

Reply:
added back the update logic to the implied "set" command